### PR TITLE
KIALI-1162 Missing warning icon on tab under ServiceDetails

### PR DESCRIPTION
**Pull request by aljesusg**
JIRA: [KIALI-1162](https://issues.jboss.org/browse/KIALI-1162)
screenshot from 2018-07-11 21-00-48

@lucasponce I changed the note **Not all checks passed** too, I think that this should show if there is an error or warning too.

### CODE REVIEW

File: src/pages/ServiceDetails/ServiceInfo.tsx [hunk 1]
```diff
@@ -130,11 +140,11 @@ class ServiceInfo extends React.Component<ServiceDetails, ServiceInfoState> {
                     <NavItem eventKey={2}>{'Source Services (' + Object.keys(dependencies).length + ')'}</NavItem>
                     <NavItem eventKey={3}>
                       {'Virtual Services (' + virtualServices.length + ')'}
-                      {validationChecks.hasVirtualServiceChecks ? errorIcon : undefined}
+                      {validationChecks.hasVirtualServiceChecks ? getValidationIcon() : undefined}
                     </NavItem>
                     <NavItem eventKey={4}>
                       {'Destination Rules (' + destinationRules.length + ')'}
-                      {validationChecks.hasDestinationRuleChecks ? errorIcon : undefined}
+                      {validationChecks.hasDestinationRuleChecks ? getSeverityIcon() : undefined}
```
Comment (lucasponce) [inline src/pages/ServiceDetails/ServiceInfo.tsx]: DestinationRules might have warnings as well.
Not today, but in next step, so I think this logic should be generic for both.

Comment (aljesusg) [reply src/pages/ServiceDetails/ServiceInfo.tsx]: So should we prepare the code or we should wait until next update?

Comment (lucasponce) [reply src/pages/ServiceDetails/ServiceInfo.tsx]: Instead to have a getValidationIcon() and getSeverityIcon() for the same task.
I would unify it now to make it generic.
Indeed, previous logic was generic, but it didn't consider 'warning' case.

Comment (xeviknal) [reply src/pages/ServiceDetails/ServiceInfo.tsx]: why are you using `getValidationIcon()` for virtual services and `getSeverityIcon()` for DestinationRules? what is the purpose?

**Comment by aljesusg** [comment]
Changed to a generic method

**Pull request merged**